packages/manager/modules/storages/src/netapp/netapp.controller.js

```diff
@@ -0,0 +1,14 @@
+export default class NetappCtrl {
+  /* @ngInject */
+  constructor($q, $translate, coreConfig, NetappService, CucRegionService) {
+    this.$q = $q;
+    this.coreConfig = coreConfig;
+    this.$translate = $translate;
+    this.NetappService = NetappService;
+    this.CucRegionService = CucRegionService;
+  }
+
+  $onInit() {
+    this.guideUrl = 'https://docs.ovh.com/fr/storage/';
```

Comment:
@JeremyDec I'm not sure adding the guide url to `core` is relevant. However having a constant file and a mapping between subsidiary and guide url could be useful (an example can be found here https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/billing/autoRenew/autorenew.constants.js#L1)

Reply:
@marie-j From my point of view, we already store guides which urls are subsection of `https://docs.ovh.com/fr/storage/`, and so it would be weird, that this module stores the main route of storage section, while other subroutes resides in our `core` module.

Reply:
I will use core constant without set a new one, the url already exist. 
I will use subsidiary to map url with